### DISCUSSION TITLE
Switching servers

### DISCUSSION THREAD

**Discussion by TheNuggetEater05** [PrismarineJS/mineflayer#2249] [Q&A]
how do i switch servers without creating 2 or more bot instances

**Comment by WhoTho** [comment]
you cant :(

**Comment by rom1504** [comment]
this is not a feature of the minecraft protocol. You need to reconnect to do that, so recreating your bot instance is the best bet since you anyway need to redo all actions on the new server

**Comment by tk746** [comment]
you cant change servers like that you need to put the ip in your createbot
